RoboFile.php

```diff
@@ -142,13 +145,74 @@ public function testCli()
             ->run();
     }
 
+    private function installDependenciesForPhp54()
+    {
+        $dependencies = [
+            'guzzlehttp/guzzle' => '5.*',
+            'symfony/finder' => '2.8.*',
+            'symfony/console' => '2.8.*',
+            'symfony/event-dispatcher' => '2.8.*',
+            'symfony/yaml' => '2.8.*',
+            'symfony/browser-kit' => '2.8.*',
+            'symfony/css-selector' => '2.8.*',
+            'symfony/dom-crawler' => '2.8.*',
+        ];
+
+        $this->installDependencies($dependencies);
+    }
+
+    private function installDependenciesForPhp55()
+    {
+        $dependencies = [
```

Comment:
I'd like not to define current dependencies here but use data from `composer.json`.
In current case dependencies are overwritten by those settings from Robofile which is not good and makes releases unpredictable.

I'd recomment to copy current `composer.json`, build php 5.4 version, restore `composer.json` and continue;